### PR TITLE
Verify membership

### PR DESCRIPTION
**Pull request by pionize**
This PR creates:

POST /verify-membership

Postman link: https://www.getpostman.com/collections/57e5477f1b0c9338a079

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -148,6 +148,7 @@ app.use(validatorMiddleware);
  */
 app.get('/', homeController.index);
 app.get('/membership', membershipController.index);
+app.post('/verify-membership', membershipController.verifyMembership);
```
Comment (haydarai) [inline app.js]: consider using `/membership/verify` endpoint

File: controllers/membership.js [hunk 2]
```diff
@@ -10,3 +15,76 @@ exports.index = (req, res) => {
...
+        if (memberId.match(/^(ST).+/) || memberId.match(/.+(S|J1|J2)$/)) {
+          isValid = false;
+        }
+        if (isValid) message = 'Member is allowed';
+
+        return res.json({
+          status: isValid,
+          message,
+          warning,
+          data: member,
```
Comment (haydarai) [inline controllers/membership.js]: Can you give another property let's say `reason`? So I can know what causes `Membership is not allowed`.

Comment (pionize) [reply controllers/membership.js]: Should we add another property or change the use of message property? I mean I will change the message property to describe the reason.

Comment (haydarai) [reply controllers/membership.js]: That works too as we have `status` that represent if it's allowed or not.

**Comment by haydarai** [comment]
@pionize when expiry date is 0000-00-00 then that means the membership is not going to be expired. Sorry, I forgot to tell you this before 😔

**Pull request merged**